website/content/en/docs/reference/vrl/_index.md

```diff
@@ -45,22 +45,35 @@ Let's have a look at a more complex example. Imagine that you're working with
 HTTP log events that look like this:
 
 ```text
-"{\"status\":200,\"timestamp\":\"2021-03-01T19:19:24.646170Z\",\"message\":\"SUCCESS\",\"username\":\"ub40fan4life\"}"
+{
+  "message": "{\"status\":200,\"timestamp\":\"2021-03-01T19:19:24.646170Z\",\"message\":\"SUCCESS\",\"username\":\"ub40fan4life\"}"
+}
 ```
 
 You want to apply these changes to each event:
 
 - Parse the raw string into JSON
-- Reformat the `time` into a UNIX timestamp
+- Attempts to convert the timestamp and checks if the conversion was successful
```

Comment:
```suggestion
- Attempt to convert the timestamp and checks if the conversion was successful
```

Reply:
This suggestion only applies if the user is the one doing this. If it's the program, then "attempts" makes sense. However, I would suggest adding that to the beginning of the sentence, e.g., "The program attempts" or "Vector attempts", whichever is more accurate.